backend/src/main/resources/application.properties

```diff
@@ -80,3 +82,7 @@ quarkus.scheduler.metrics.enabled=true
 # Sources integration URLs and details. It is used to store the secrets' data for the camel and webhook endpoints.
 quarkus.rest-client.sources.read-timeout=1000
 quarkus.rest-client.sources.url=${clowder.endpoints.sources-api-svc:http://localhost:8000}
+
+# OpenTelemetry -- see also jdbc driver above.
+quarkus.opentelemetry.enabled=false
+quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317
```

Comment:
Will the OTLP host and port be provided by Clowder eventually?

Reply:
Yes, eventually.